ida-partner.cer

```diff
@@ -0,0 +1,22 @@
+-----BEGIN CERTIFICATE-----
+MIIDrDCCApSgAwIBAgIIkcTLqBQn22gwDQYJKoZIhvcNAQELBQAwdjELMAkGA1UE
```

Comment:
Why do we store a ida-partner.cer in config? I suggest to remove it, if needed take it from ida-internal service's getCertificate endpoint.

Reply:
removed the file